0_intro/exercises/cat/src/main.rs

```diff
@@ -1,7 +1,14 @@
+use std::env;
+use std::io::Read;
+
 pub fn cat(path: &str) -> String {
-    todo!();
+    let mut file = std::fs::File::open(path).unwrap();
+    let mut contents = String::new();
+    file.read_to_string(&mut contents).unwrap();
+    contents
 }
 
 fn main() {
-    todo!();
+    let args: Vec<String> = env::args().collect();
+    println!("{}", cat(&args[1]));
```

Comment:
Careful, this will panic if no argument is given to the program. It won't cause a crash or a segfault, but you will trigger the bounds checker. You should use a different method to access the first element and check if it is present or not. If not present, then cat shouldn't do anything, but it shouldn't crash